src/components/Button.tsx

```diff
@@ -13,12 +13,20 @@ export interface ButtonProps {
 	style?: object; // may be unsafe, but this is the type provided
 	// by Stylesheet documentation:
 	// https://reactnative.dev/docs/stylesheet#compose
+	children?: JSX.Element;
```

Comment:
```suggestion
	children?: Array<JSX.Element>;
```

I think I remember needing this to be an array for a component I made while working but I could be misremembering

Reply:
Actually technically the right thing to do is use React.FC<{other props}> and it'll add in children with the correct type, i guess I was just being lazy and didn't want to change what was already there but i'll fix it